### DISCUSSION TITLE
Logo not showing

### DISCUSSION THREAD

**Discussion by checksumit** [baimard/gestion#270] [Maybe a bug ?] [answered]
Hi, firstly, thank you for the great application it's quality for sure and very much needed. 

I'm trying to get the logo to show on the invoice. I have moved the logo.png to the gestion folder, but it's still not showing.

Are you able to assist or anyone else?
![image](https://user-images.githubusercontent.com/59392287/185025964-87f4cbd9-6a47-4250-8aa2-06f3e6527533.png)

**Comment by checksumit** [comment, marked as answer]
Got it working!!!! 👍

**Reply by baimard** [reply]
Like this kind of bug ahah.

Maybe you ve got a cache problem ?

**Reply by baimard** [reply]
Just FYI logo need to be placed in the nextcloud folder with your profil, not on the system file !

**Reply by checksumit** [reply]
Thank you, yes... Not the system file!... I was looking far too deep into it!

**Comment by denisv29ma** [comment]
Bonjour,
mon logo ne s'affiche pas non plus, je pense avoir testé toutes les solutions trouvées içi, mais rien ne marche !!
J'ai créé le dossier .gestion avec les bons droits dans apps/gestion, j'ai testé en fichier non caché, j'ai mis dans /user/files/ pareil je m'arrache les cheveux !!
Merci de votre aide
